sw/device/lib/dif/dif_aes_unittest.cc

```diff
@@ -190,5 +190,37 @@ TEST_F(DecryptTest, start) {
   EXPECT_EQ(dif_aes_start(&aes_, &transaction, kKey_, NULL), kDifOk);
 }
 
+// Key size test
+class Key192Test : public AesTestInitialized {
+ protected:
+  Key192Test() { transaction.key_len = kDifAesKey192; }
+};
+
+TEST_F(Key192Test, start) {
+  EXPECT_READ32(AES_STATUS_REG_OFFSET, 1);
+  setExpectedConfig(AES_CTRL_SHADOWED_KEY_LEN_VALUE_AES_192,
+                    AES_CTRL_SHADOWED_MODE_VALUE_AES_ECB,
```

Comment:
The Ideia here is to cover all the possible fields in the register, so a combination with this key size with different modes would not increase the coverage. Althought it would make sence for a chip level test indeed.